pages/activate/[uid]/[token]/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { appGetLayout } from "components/AppRoute";
+import Signup from "features/auth/signup/Signup";
+import { GetStaticPaths, GetStaticProps } from "next";
+import nextI18nextConfig from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+export const getStaticPaths: GetStaticPaths = () => ({
+  paths: [],
+  fallback: "blocking",
+});
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(
+      locale ?? "en",
+      ["global", "auth"],
+      nextI18nextConfig
+    )),
+  },
+});
```

Comment:
```suggestion
// Further above...
import { AUTH, GLOBAL } from "i18n/namespaces";
import { translationStaticProps } from "i18n/server-side-translations";

export const getStaticProps = translationStaticProps([GLOBAL, AUTH]);
```

There's a bit of tech debt with this where some places of the codebase is still doing what this `translationStaticProps` helper function has nicely abstracted away, and so should be preferred when creating new pages level components/files!